packages/xdl/src/start/startAsync.ts

```diff
@@ -50,6 +52,32 @@ export async function startAsync(
   verbose: boolean = true
 ): Promise<ExpoConfig> {
   assertValidProjectRoot(projectRoot);
+
+  const userData = await UserManager.getCachedUserDataAsync();
+
+  if (userData?.userId) {
+    // analytics has probably not identified the user at this point, if so we need to bootstrap it
+    if (!UnifiedAnalytics.userId) {
+      UnifiedAnalytics.identifyUser(
+        userData.userId, // userId is used as the identifier in the other codebases (www/website) running unified analytics so we want to keep using it on the cli as well to avoid double counting users
+        {
+          userId: userData.userId,
+          currentConnection: userData?.currentConnection,
+          username: userData?.username,
+          userType: '', // not available without hitting api
+        }
+      );
+    }
+  }
```

Comment:
Every new event will still have the same problem, it's ok as tmp fix, but the solution should be more generic

you could move it to function in `packages/xdl/src/UnifiedAnalytics.ts` and call that function from `packages/expo-cli/src/exp.ts` after setVersionName

Reply:
since we only need the start event for now I'm keeping it at this level, but do plan to pull it up if we have more events